docs/src/data/sample_config.js

```diff
@@ -21,6 +21,27 @@ const config = {
           "src/auth/middleware/auth_middleware.rb:45": "Add documentation for this class"
         }
       },
+      "additional_metrics": {
```

Comment:
The JSON payload generated by the Ruby gem and handed off to the documentation site seem to represent a contract between these two applications. The more content we add into this payload, the tighter it feels like these two application are coupled together. I worry that this will eventually result in maintenance challenges and regression bugs (e.g. we change a seemingly internal aspect of the cyclomatic complexity calculation and the documentation site fails to load).

This is not something we can/should address in this PR specifically, but as we consider further changes that would extend the contents of this payload, we should look for ways to isolate the contract that this file represents within the ruby gem to ensure it is clearly defined and able to be effectively maintained.